packages/web-components/fast-foundation/src/text-field/text-field.ts

```diff
@@ -218,15 +219,15 @@ export class TextField extends FormAssociated<HTMLInputElement> {
         if (this.proxy instanceof HTMLElement) {
             this.proxy.value = this.value;
         }
```

Comment:
```suggestion
        }
       if (this.control) {
            this.control.value = this.value;
        }
```

Because we've got both a proxy input and a control input, we need to update both when the value changes. Otherwise, things can get out of sync. I experience this bug while preparing my demos for day of learning and had to hack this fix locally to get things working correctly.

Reply:
added this suggestion